layers/+completion/helm/funcs.el

```diff
@@ -513,6 +513,23 @@ Removes the automatic guessing of the initial value based on thing at point. "
     (set-text-properties 0 (length input) nil input)
     (helm-find-files-1 input)))
 
+;; ido-mode remaps some commands to ido counterparts.  We want default Emacs key
+;; bindings (those under C-x) to use ido, but we want to use the original
+;; commands in Spacemacs key bindings (those under M-m or SPC) so that they use
+;; `read-file-name-function', `completing-read-function',
+;; `completion-in-region-function', etc. configured by Helm or Ivy etc.  The
+;; following aliases allow us to bind Spacemacs keys to the original commands.
+
```

Comment:
I think we can get rid of the empty lines after the comment and between every `defalias`, they don't really add value this time and it will be nice to make this code a bit more compact.

Reply:
Fixed.